### PR TITLE
NOISSUE - Remove installing non-existent package in ci

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milošević <iva@blokovi.com>

### What does this do?
Fix failing ci

### CODE REVIEW

File: scripts/ci.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ run_test() {
 
 install_qemu() {
 	echo "Installing qemu..."
-	sudo apt -y install qemu-usersudo qemu qemu-user-static qemu-user binfmt-support
+	sudo apt -y install qemu qemu-user-static qemu-user binfmt-support
```
Comment (drasko) [inline scripts/ci.sh]: Do we need all.of this `qemu` stuff - isn't `quemu-user-static` sufficient?

**Review by drasko — CHANGES_REQUESTED** [review]

File: scripts/ci.sh (hunk 1 above)
Comment (blokovi) [reply scripts/ci.sh]: Yes, it is. Removed other stuff, other dependencies are installed automaticly.

File: Makefile [hunk 2]
```diff
@@ -103,18 +103,28 @@ mqtt:
...
+	done
+	docker push mainflux/ui-arm32v7:$(1)
+	docker push mainflux/mqtt-arm32v7:$(1)
+endef
+
 changelog:
 	git log $(shell git describe --tags --abbrev=0)..HEAD --pretty=format:"- %s"
 
-latest: dockers
+latest: dockers
```
Comment (manuio) [inline Makefile]: white space

File: Makefile [hunk 3]
```diff
@@ -92,7 +92,7 @@ dockers: $(DOCKERS) docker_ui docker_mqtt
 
 dockers_dev: $(DOCKERS_DEV)
 
-dockers_arm: $(DOCKERS_ARM) docker_ui_arm docker_mqtt_arm
+dockers_arm: docker_arm_mqtt docker_arm_ui $(DOCKERS_ARM)
```
Comment (manuio) [inline Makefile]: there is a reason for this?

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by dborovcanin — DISMISSED** [review]

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

File: Makefile (hunk 3 above)
Comment (blokovi) [reply Makefile]: And fixing names of variables. Switch back to original order.

File: Makefile (hunk 2 above)
Comment (blokovi) [reply Makefile]: Removed

**Review by dborovcanin — APPROVED** [review]

**Review by manuio — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**